eslint.config.js

```diff
@@ -11,14 +11,21 @@ module.exports = {
     jest: true,
   },
   rules: {
-    'max-len': 'off', // Remove warnings on max line length exceeding 100 characters
-    'react/require-default-props': 'off', // Disabled the requirement to default all non-required props
+    // Remove warnings on max line length exceeding 100 characters
+    'max-len': 'off',
+    // Disabled the requirement to default all non-required props
+    'react/require-default-props': 'off',
```

Comment:
Did we want to start reenabling this rule?

Reply:
We had a discussion in the war room a long time ago to reenable this rule at some point.

My personal preference is to leave this rule off, but if we want to enforce defaulting props as a standard it makes sense to reenable it.